apps/avatax/e2e/data/maps/product.json

```diff
@@ -9,6 +9,11 @@
       "variantId": "UHJvZHVjdFZhcmlhbnQ6Mzg0",
       "id": "UHJvZHVjdDoxNTI=",
       "name": "Apple Juice"
+    },
+    "Shirt": {
+      "variantId": "UHJvZHVjdFZhcmlhbnQ6MzU4",
+      "id": "UHJvZHVjdDoxMzY=",
+      "name": "Darko Polo"
```

Comment:
@szczecha We should update our snapshot because we've updated product variant and added new voucher

Reply:
Done :) I cleaned snapshots a bit. I added the voucher and promotions and assigned more products to the channel. 
We need to figure out how to handle development it should not be by default on the same sandbox as the sandbox for testing release. However, there is a problem with restoring snapshots on another sandbox now we need to reinstall and reconfigure the app.